### PR TITLE
Chess-12

### PR DESCRIPTION
**Pull request by Avrey125**
Basic app structure built

### CODE REVIEW

File: chess-game/src/components/piece/basicPieces.jsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ const Pieces = () => {
       let x = index * width/array.length + spaceWidth/2;
       function movement(e) {
         x = x + 75; 
-        console.log('hhello')
+        console.log('hello')
         return x
       }
       items.push(
```
Comment (RatioChris) [inline chess-game/src/components/piece/basicPieces.jsx]: I'm seeing an error in this loop. Put the key on the wrapper element, not the children:
`<g key={index}>`
You can remove the keys from Piece.

**Comment by RatioChris** [comment]
A couple comments:
- No need for the `chess-game` base directory.  Just move everything in there to the root.
- I'm seeing a couple linting errors in the terminal.

**Comment by Avrey125** [comment]
Fixed problem, repushed

File: chess-game/src/components/piece/basicPieces.jsx (hunk 1 above)
Comment (Avrey125) [reply chess-game/src/components/piece/basicPieces.jsx]: Problem fixed, repushed

**Comment by elrondshep** [comment]
In future PRs - can you add to the title a brief something about the topic. You got the Jira number in there - perfect! Maybe "Chess-12 - Update app structure". Something small that gives an idea at a glance what Chess-12 is. For more details people can go to Jira. Branches should be named similarly - chess-12-update-app-structure. It's hard to keep it small sometimes.

**Pull request merged**